Modules/Relationships/src/Relationships.Domain/DomainErrors.cs

```diff
@@ -41,9 +41,9 @@ public static DomainError RelationshipToTargetAlreadyExists(string targetIdentit
             $"A relationship to '{targetIdentity}' already exists. If the relationship is terminated, you can reactivate it.");
     }
 
-    public static DomainError NoOpenReactivationRequest(string targetIdentity)
+    public static DomainError NoOpenReactivationRequest(string activeIdentity)
     {
         return new DomainError("error.platform.validation.relationshipRequest.noOpenReactivationRequest",
-            $"There is no open reactivation request for '{targetIdentity}', and only '{targetIdentity}' can revoke a relationship reactivation.");
+            $"There is no open reactivation request for '{activeIdentity}', and only '{activeIdentity}' can revoke a relationship reactivation.");
```

Comment:
Similar as above. The error code (or method name) does not say anything about revocation. So one might use it in any scenario (accept, reject, revoke). But in the error message, you talk about "revoke".

Reply:
Changes made.